src/main/kotlin/no/fdk/dataservicecatalog/handler/RDFHandler.kt

```diff
@@ -223,6 +223,12 @@ fun Model.addDataService(dataService: DataService, dataServiceUri: String) {
         )
     }
 
+    dataService.status?.takeIf(String::isNotBlank)?.let {
```

Comment:
Lurer på om denne er unødvendig å ha med. Vil populeres i henhold til kva som er brukt i grafen, virker det som.

`
setNsPrefixes(
    mapOf(
        "dcat" to DCAT.NS,
        "dct" to DCTerms.NS,
        "rdf" to RDF.uri,
        "vcard" to VCARD4.NS,
        "foaf" to FOAF.NS
    )
)
`

Viss ikkje, må den oppdateres med ADMS.

Reply:
Må nok ha med, eneste snarveien jeg kjenner til er å kopiere prefixer fra andre grafer

Reply:
7ec448e30309bfdeb704aa0c0bc51c77b3a462f9